cristin-project/src/test/java/no/unit/nva/cristin/projects/query/QueryCristinProjectHandlerTest.java

```diff
@@ -643,6 +642,40 @@ void handlerThrowsBadRequestWhenStatusQueryParamsIsInvalid() throws IOException
                                                                             Arrays.toString(ProjectStatus.values()))));
     }
 
+    @Test
+    void shouldAddLangToHttpRequestBeforeSendingToCristin() throws Exception {
+        var cristinProjects = IoUtils.stringFromResources(Path.of(CRISTIN_QUERY_PROJECTS_RESPONSE_JSON));
+        var oneCristinProject = IoUtils.stringFromResources(Path.of(CRISTIN_GET_PROJECT_RESPONSE_JSON));
+        HttpResponse<String> queryResponse = new HttpResponseFaker(cristinProjects, HTTP_OK);
+        HttpResponse<String> getResponse = new HttpResponseFaker(oneCristinProject, HTTP_OK);
+        var mockHttpClient = mock(HttpClient.class);
+        when(mockHttpClient.<String>send(any(), any())).thenReturn(queryResponse);
+        when(mockHttpClient.<String>sendAsync(any(), any())).thenReturn(CompletableFuture.completedFuture(getResponse));
+        var apiClient = new QueryCristinProjectApiClient(mockHttpClient);
```

Comment:
easier to read 
`        var apiClient = spy( new QueryCristinProjectApiClient(mockHttpClient));
` ?